lib/simplecov.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "English"
+require_relative "simplecov/process"
```

Comment:
Not sure requiring this by default is our best option. While I'm all for providing a good out of the box experience, I also don't want to freedom patch core - I see that it seems necessary for what you want to achieve unless we'd put more burden on the user. What do you think about telling users themselves to require this? (+ I'd probably name it `process_patch` or something to indicate what we're doing)

Reply:
I was thinking of enabling it with a config like `SimpleCov.enable_for_forks` or something with documentation about how that modifies `Process.fork`, and conditionaly requiring it in SimpleCov.start. what do you think?

Reply:
Hm unsure. I guess I could go either way on it. In your mind, what'd be the advantage of that over telling people to do `require "simplecov/something"` ?

The upside I see is that it gives us a place to do more stuff should we need it. The downside is that it's more complex.

:man_shrugging:

Reply:
nothing but that it feels like configuration